### ISSUE TITLE
maven install error jaws

### ISSUE THREAD

**Issue by juandm** [AgreementMakerLight/AML-Project#8]
Hi there,

Im triying to use the software but the `maven install` command failed resolving the jaws dependency:

```sh
[ERROR] Failed to execute goal on project aml: Could not resolve dependencies for project aml:aml:jar:2.1-SNAPSHOT: 

Failed to collect dependencies at edu.smu.tspell.wordnet:jaws:jar:1.3.3: 

Failed to read artifact descriptor for edu.smu.tspell.wordnet:jaws:jar:1.3.3: 

Could not transfer artifact edu.smu.tspell.wordnet:jaws:pom:1.3.3 from/to ironiacorp (http://www.ironiacorp.com/repo/maven2):

 www.ironiacorp.com: Unknown host www.ironiacorp.com
```

I googled this repo but i was unable to find it.

Thanks in advance

**Comment by AgreementMakerLight** [comment]
Hi!
Yes, the JAWS library that AML uses with WordNet is no longer available through Maven.
Although far from ideal, the solution we've adopted for the time being is to include AML's dependencies in the git repository. Thus, if you clone/pull the AML-Project you'll find all the dependencies in the bin/lib folder. You can add them manually to the classpath (e.g., using Eclipse) and just ignore the Maven build or you can alter the Maven pom to use the local JAWS library in the bin/lib.
If you just want AML in jar form, you can also get it from https://github.com/AgreementMakerLight/AML-Jar.
Best

**Comment by juandm** [comment]
Thank you for the response!
